### PR TITLE
[cdoc] introduction

### PR DESCRIPTION
**Pull request by YitongFeng**

### CODE REVIEW

File: webportal/README-zh.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### 查看集群状态
+点击 “Cluster View”查看整个集群的状态，包括：
+
+* 服务: 每台机器上所有服务的状态
+* 硬件: 每台机器的硬件信息
+* Kubernetes 页面
+
+### 查阅文档
+点击“Documents”可以查阅提交 job 及下载 `pai-fs` 工具等教程。
```
Comment (abuccts) [inline webportal/README-zh.md]: #599 , pls update accordingly

Comment (YitongFeng) [reply webportal/README-zh.md]: Solved

File: README-zh.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Open Platform for AI (PAI) ![alt text][logo]
+
+[logo]: ./pailogo.jpg "OpenPAI"
+
+[![Build Status](https://travis-ci.org/Microsoft/pai.svg?branch=master)](https://travis-ci.org/Microsoft/pai)
+[![Coverage Status](https://coveralls.io/repos/github/Microsoft/pai/badge.svg?branch=master)](https://coveralls.io/github/Microsoft/pai?branch=master)
+
+
+## 概述
```
Comment (fanyangCS) [inline README-zh.md]: >概述 [](start = 3, length = 2)

please exclude the top level readme from this PR and focus on webportal only. Also, the readme has been revised, please update the translation accordingly.

Comment (YitongFeng) [reply README-zh.md]: OK, and I noticed your update, this is already new translation.

**Pull request closed without merging**